src/librustc/middle/region.rs

```diff
@@ -8,22 +8,17 @@
 
```

Comment:
The `This file builds up the ScopeTree` comment is now outdated.

Reply:
Maybe just change it to `This file declares ScopeTree`? The rest is still relevant.